tests/Unit/NumericalAlgorithms/FiniteDifference/Test_HighOrderFluxCorrection.cpp

```diff
@@ -307,9 +504,29 @@ SPECTRE_TEST_CASE("Unit.FiniteDifference.CartesianHighOrderFluxCorrection",
   for (const fd::DerivativeOrder correction_order :
        {DO::Two, DO::Four, DO::Six, DO::Eight, DO::Ten, DO::OneHigherThanRecons,
         DO::OneHigherThanReconsButFiveToFour}) {
-    test<1>(correction_order);
-    test<2>(correction_order);
-    test<3>(correction_order);
+    CAPTURE(correction_order);
+    test<1>(correction_order, true);
+    test<2>(correction_order, true);
+    test<3>(correction_order, true);
+    test<2>(correction_order, false);
+  }
+  // Test that error improves as we increase the correction order. This part
+  // could be improved because, currently, increasing the correction order
+  // also increases the number of grid points in the test, as well as the
+  // maximum order of the polynomial that the flux/divergence functions are
+  // dependent on. There is still an improvement in fractional error with
+  // correction order, but it's hard to necessarily isolate the effect of
+  // the finite difference derivative order from these other effects as the test
+  // currently stands.
```

Comment:
Between this comment and the `2e-2` tolerance, I'm currently 70% convinced you have a bug :P